### ISSUE TITLE
Support lists of songs (both imported from the cloud or entered by the user)

### ISSUE THREAD

**Issue by waldyrious** [TeamOfOne/psc16-team33#30]

**Comment by waldyrious** [comment]
Bonus: various sorting / filtering / searching methods: by author, by length, by difficulty, by style, by popularity...

**Comment by elincognito** [comment]
For the cloud i suggest using [firebase](https://www.firebaseio.com/), its simple to use and setup.
Also it allows for realtime updates, 
If in the feature we decide for instance to allow two users to simultaneous mess around in the same  song the updates are pushed "instantly" to each other.

**Comment by waldyrious** [comment]
Agreed, and in fact I've been meaning to use firebase in a project for quite a while:)

**Comment by waldyrious** [comment]
Here's a nice UI/UX that we can take inspiration from: https://play.riffstation.com/?v=OPf0YbXqDm0

**Comment by fjsnogueira** [comment]
I've started a branch that adds the support for list of songs created by the user, I've attached some files of the screenshots:

![192 168 1 75-8100- iphone 6 8](https://cloud.githubusercontent.com/assets/1634910/18608763/1c7e512e-7cea-11e6-8e54-e3ec7f6fd29d.png)
![192 168 1 75-8100- iphone 6 9](https://cloud.githubusercontent.com/assets/1634910/18608766/1c7ef8d6-7cea-11e6-82e6-8ff386eb29df.png)
![192 168 1 75-8100- iphone 6 10](https://cloud.githubusercontent.com/assets/1634910/18608762/1c7de978-7cea-11e6-9feb-e1c231f79b17.png)
![192 168 1 75-8100- iphone 6 11](https://cloud.githubusercontent.com/assets/1634910/18608764/1c7e5da4-7cea-11e6-96f5-b861bddf5ec1.png)

**Comment by fjsnogueira** [comment]
I have just updated the screenshots to reflect last commit (aligning to center the empty playlist warning)

**Comment by waldyrious** [comment]
Looking good! :) Just a slight note: I'd use an information icon, rather than a warning one, to match the severity (low) of the corresponding message.
